DigitalOffice/Tests/UserServiceTests/Commands/EditUserCommandTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using FluentValidation;
+using Moq;
+using NUnit.Framework;
+using System;
+using LT.DigitalOffice.UserService.Commands;
+using LT.DigitalOffice.UserService.Commands.Interfaces;
+using LT.DigitalOffice.UserService.Database.Entities;
+using LT.DigitalOffice.UserService.Mappers;
+using LT.DigitalOffice.UserService.Mappers.Interfaces;
+using LT.DigitalOffice.UserService.Repositories.Interfaces;
+using LT.DigitalOffice.UserService.RestRequests;
+using LT.DigitalOffice.UserService.Validators;
+
+namespace LT.DigitalOffice.UserServiceUnitTests.Commands
+{
+    class EditUserCommandTests
+    {
+        private Mock<IUserRepository> repositoryMock;
+        private IEditUserCommand command;
+        private IValidator<EditUserRequest> validator;
+        private IMapper<EditUserRequest, DbUser> mapper;
+
+        [SetUp]
+        public void Initialization()
+        {
+            repositoryMock = new Mock<IUserRepository>();
+            mapper = new EditUserRequestToDbUserMapper();
+            validator = new EditUserRequestValidator();
+
+            command = new EditUserCommand(validator, repositoryMock.Object, mapper);
+        }
+
+        [Test]
+        public void ShouldThrowExceptionWhenUserDataIsInvalid()
+        {
+            var request = new EditUserRequest
+            {
+                Id = Guid.NewGuid(),
+                FirstName = "Example1",
+                LastName = "Example",
+                MiddleName = "Example",
+                Email = "Example@gmail.com",
+                Status = "Example",
+                Password = "Example",
+                IsAdmin = false
+            };
+
+            Assert.Throws<ValidationException>(() => command.Execute(request));
```

Comment:
It would be cool to also check that the repositoryMock method did not get called

Reply:
Fixed

Reply:
Not fixed :(

Reply:
Hmm, strange. The github repository didn't catch my changes :(

Reply:
Fixed